contrib/fuzz/oss_fuzz_build.sh

```diff
@@ -37,6 +37,10 @@ compile_fuzzers() {
     done
 }
 
+cd $SRC/
+git clone https://github.com/AdamKorcz/go-fuzz-headers-1 --depth=1
```

Comment:
While it is in contrib/ directory, we are running this script for each PR and making this reproducible is important. We cannot let this script break even if the tip of go-fuzz-headers-1 is broken.

Can you merge the change in go-fuzz-headers and then upgrade our go.mod?

Reply:
Atm I would prefer to not merge the changes into upstream go-fuzz-headers as it would break things for other projects. What about pinning https://github.com/AdamKorcz/go-fuzz-headers-1 to a specific commit for reproducibility?

Reply:
Replaced `github.com/AdaLogics/go-fuzz-headers` with `github.com/AdamKorcz/go-fuzz-headers-1` in `go.mod` and `vendor/`